components/AlertsList.vue

```diff
@@ -6,10 +6,25 @@
     >
       {{ $t("index.massSite") }}
     </alert>
+    <alert
+      :buttonText="$t('index.getDocument')"
+      buttonUrl="https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/vaccine_tagengo.html"
+    >
+      {{ $t("index.prevaccinationScreening") }}
+    </alert>
   </v-container>
 </template>
 <style>
 .alerts-list {
   margin: 0.5em 0 2em 0;
 }
+
+@media only screen and (max-width: 600px) {
```

Comment:
Can we try Vuetify breakpoints instead?

https://v15.vuetifyjs.com/en/framework/breakpoints